pkgs/development/interpreters/rakudo/moarvm.nix

```diff
@@ -3,12 +3,12 @@
 
 stdenv.mkDerivation rec {
   pname = "moarvm";
-  version = "2020.01.1";
+  version = "2020.02";
 
   src = fetchurl {
-    url    = "https://github.com/MoarVM/MoarVM/releases/download/${version}/MoarVM-${version}.tar.gz";
-    sha256 = "11rmlps6r3nqa9m2yyv9i2imahirsqmxbfay71f3gs4ql121xdnw";
-  };
+    url = "https://www.moarvm.org/releases/MoarVM-2020.02.tar.gz";
```

Comment:
Shouldn't the `${version}` be embedded here? WAS THIS A COPY?PASTE ACCIDENT?

Reply:
:+1: